bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/CoordinateSystemMapper.java

```diff
@@ -25,7 +25,7 @@ interface CoordinateSystemMapper {
 
 	Point map(Control from, Control to, int x, int y);
 
-	Rectangle mapMonitorBounds(Rectangle rectangle, int zoom);
+	Rectangle mapMonitorBounds(Rectangle rectangle);
```

Comment:
Why not as follows to avoid unnecessary type checks?
```suggestion
	Rectangle mapMonitorBounds(Rectangle.WithMonitor rectangle);
```